src/vs/workbench/contrib/editSessions/browser/editSessions.contribution.ts

```diff
@@ -158,11 +158,11 @@ export class EditSessionsContribution extends Disposable implements IWorkbenchCo
 				title: EDIT_SESSIONS_TITLE,
 				ctorDescriptor: new SyncDescriptor(
 					ViewPaneContainer,
-					[EDIT_SESSIONS_CONTAINER_ID, { mergeViewWithContainerWhenSingleView: true }]
+					[EDIT_SESSIONS_CONTAINER_ID, { mergeViewWithContainerWhenSingleView: true, donotShowContainerTitleWhenMergedWithContainer: true }]
```

Comment:
camel casing for `donot`? 😅 🐫

Reply:
Yeah that bothers me too, I think @sandy081 might be the owner of the views code, I can send a PR 🙂